pywiscat/wis1/report.py

```diff
@@ -141,5 +145,83 @@ def records_by_org(ctx, directory, file_list_file, group_by_authority, verbosity
         click.echo('No results')
 
 
+@click.command()
+@click.pass_context
+@cli_callbacks
+@click.option('--directory', '-d', required=False,
+              help='Directory with metadata files to process',
+              type=click.Path(resolve_path=True, file_okay=False))
+@click.option('--file-list', '-f', 'file_list_file',
+              type=click.Path(exists=True, resolve_path=True), required=False,
+              help='File containing JSON list with metadata files to process, alternative to "-d"')
+@click.option('--kpi', '-k', default=0, help='KPI to run, default is all')
+@click.option('--summary', '-s', is_flag=True, default=False,
```

Comment:
Are `--summary` and `--brief-summary` exclusive?  Are we able to combine them into a single option called `elementset=brief|summary` (this is similar to CSW search/present semantics fwiw).

Reply:
`--summary` outputs the "Summary" section of the pywcmp output for each file. That seemed to me still quite verbose, so I have added `--brief-summary` option which is the output that I posted in the issue. In other words, it is exclusive, so it could be instead combined into one option, let's say `--output-format` with three possible values: "full, summary, brief-summary". I would say that the `brief-summary` could be the default, but I am biased.

Reply:
can we call it `brief/summary/full`?  This is consistent with OGC CSW, for example.